Tests/AbstractTests/Utility/CustomArbitraryTypes.swift

```diff
@@ -1,8 +1,5 @@
 import XCTest
 @testable import Abstract
-#if !XCODE_BUILD
```

Comment:
This is needed for the package build still, no?  You can use the positive condition `#if SWIFT_PACKAGE` and remove `XCODE_BUILD` from the project if you wish.

Reply:
You're right, I just changed everything into SWIFT_PACKAGE and removed the XCODE_BUILD flag